server/vmap/trillian_map_server.go

```diff
@@ -74,33 +78,47 @@ func (t *TrillianMapServer) GetLeaves(ctx context.Context, req *trillian.GetMapL
 
 	smtReader := merkle.NewSparseMerkleTreeReader(req.Revision, hasher, tx)
 
-	leaves, err := tx.Get(ctx, req.Revision, req.Index)
-	if err != nil {
-		return nil, err
-	}
-	glog.Infof("%v: wanted %v leaves, found %v", mapID, len(req.Index), len(leaves))
+	inclusions := make([]*trillian.MapLeafInclusion, 0, len(req.Index))
+	found := 0
+	for _, index := range req.Index {
+		// TODO: specify the index length in the tree specification.
+		if got, want := len(index), 32; got != want {
```

Comment:
(personal preference) as this isn't a test, personally I'd just do `if len(index) != indexSize`